Formula/popte2.rb

```diff
@@ -0,0 +1,37 @@
+class Popte2 < Formula
+  desc "Enables comparative population genomics of transposable elements (TE). As a major innovation
+  PoPoolation TE2 introduces the physical pileup file which allows to homogenize the power to identify
+  TEs and thus enables an unbiased comparison of TE abundance between samples, where samples could be
+  pooled populations, tissues or sequenced individuals."
+  homepage "https://sourceforge.net/projects/popoolation-te2/"
+  url "https://downloads.sourceforge.net/project/popoolation-te2/previous-releases/popte2-v1.10.04.jar"
+  sha256 "58798c37ed3ff7d5f24d44af3108af485298fbea0eba1a27e535af2e6ff1bb5a"
+
+  bottle :unneeded
+
+  depends_on :java => ["1.8", :run]
+
+  def install
+    java = share/"java"
+    java.install Dir["*.jar"]
+    bin.write_jar_script java/"popte2-v1.10.04.jar", "popte2"
+  end
+
+  def post_install
+    inreplace "#{bin}/popte2", "exec java ", "exec java ${JAVA_OPTS:--Xmx4g}"
```

Comment:
Why the default java option is `--Xmx4g`? I think that this should be just like in ReadTools (and I am not sure if all the tools in the jar file will work properly with 4GB memory...)

Reply:
This is how the doc says popte2 should be run: java -Xmx4g -jar popte2...jar

Reply:
I believe that the `--Xmx4g` is for the walkthrough data; in other parts of the manual Robert says `java -jar popte2.jar`.

In addition, setting a default option in this case is tricky, because whenever the JAVA_OPTS is overriden, the default even if you provide a different parameter: that's an inconsistency that could have unexpected results.

My recommendation: remove the default value here.